From 817dbe77e3e5a6d89540000a48584358efb4b03a Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Mon, 1 Apr 2019 07:33:19 +0200
Subject: [PATCH] rpmio/digest_nss.c: fix build on musl

signal.h must be included to be able to use sigaction

Fixes:
 - http://autobuild.buildroot.org/results/395fd44a930dfc2ad380bc735c26d9ce62344295

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status:
https://github.com/rpm-software-management/rpm/pull/650]
---
 rpmio/digest_nss.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpmio/digest_nss.c b/rpmio/digest_nss.c
index 50f8c8e90..1692a9221 100644
--- a/rpmio/digest_nss.c
+++ b/rpmio/digest_nss.c
@@ -3,6 +3,7 @@
 #include <pthread.h>
 #include <nss.h>
 #include <sechash.h>
+#include <signal.h>
 #include <keyhi.h>
 #include <cryptohi.h>
 #include <blapit.h>
-- 
2.14.1

